data.tf

```diff
@@ -1,4 +1,5 @@
-data "aws_region" "current" {}
+data "aws_region" "current" {
+}
```

Comment:
Does `terraform fmt` in 0.12 make this change?

Reply:
`terraform  0.12upgrade` does

Reply:
`terraform fmt` for `0.12.2` appears to be satisfied with either syntax. `terraform -0.12upgrade` has some unexpected opinions, like ending each file with a double line break, which `terraform fmt` also appears neutral on.